packages/mobile/src/geth/geth.ts

```diff
@@ -88,6 +88,13 @@ async function createNewGeth(): Promise<typeof RNGeth> {
     syncMode,
     useLightweightKDF: true,
     ipcPath: IPC_PATH,
+    noDiscovery: !USE_FULL_NODE_DISCOVERY,
+  }
+
+  if (USE_FULL_NODE_DISCOVERY) {
+    const nodes = (await BootnodeUtils.getBootnodesAsync(DEFAULT_TESTNET)).trim()
+    gethOptions.bootnodeEnodes = nodes.split(';')
+    Logger.debug('Geth@newGeth', 'bootnodes = ' + nodes.split(';'))
```

Comment:
Nit, consider assigning a var to nodes.split(';') so we don't do it twice.
Also we can use string interpolation here instead of the manual concatenation